### PR TITLE
Fixed incorrect Russian graphing translations

### PR DESCRIPTION
**Pull request by BuiIdinger**
## Fixes #.

https://github.com/microsoft/calculator/issues/2052
https://github.com/microsoft/calculator/issues/2053

### Description of the changes:
- Fixed Russian translations in graphing calculator
- Corrected contribute text in settings
-

### How changes were validated:

- manual
-
-

### CODE REVIEW

**Comment by BuiIdinger** [comment]
@microsoft-github-policy-service agree

**Review by team213123 — APPROVED** [review]

File: src/Calculator/Resources/ru-RU/Resources.resw [hunk 1]
```diff
@@ -3627,19 +3627,19 @@
     <comment>Error that occurs during graphing when i or I is used.</comment>
   </data>
   <data name="GeneralError" xml:space="preserve">
-    <value>Уравнение не может быть построено</value>
+    <value>График уравнения не может быть построен</value>
     <comment>General error that occurs during graphing.</comment>
   </data>
   <data name="InvalidNumberDigit" xml:space="preserve">
-    <value>Цифра не может быть разрешена для данной базы</value>
+    <value>Цифра не может использоваться с этим основанием</value>
```
Comment (STeeL835) [inline src/Calculator/Resources/ru-RU/Resources.resw:3634]:
```suggestion
    <value>Не удалось определить цифру для заданного основания</value>
```
Though I haven't found how to trigger this error to see what it is about

**Review by STeeL835 — COMMENTED** [review]
Some errors I haven't been able to check to see them in context (for example "deriv", I guess mentions of it exists in the codebase, but it wasn't actually implemented), but I mostly agree with the corrections

File: src/Calculator/Resources/ru-RU/Resources.resw [hunk 2]
```diff
@@ -3599,15 +3599,15 @@
...
+    <value>Знак равенства использован без уравнения</value>
     <comment>Error that occurs during graphing when equal is used without an equation. Ex: sin(x=y)</comment>
   </data>
   <data name="ExpectParenthesisAfterFunctionName" xml:space="preserve">
     <value>Скобки отсутствуют после имени функции</value>
     <comment>Error that occurs during graphing when parenthesis are missing after a function.</comment>
   </data>
   <data name="IncorrectNumParameter" xml:space="preserve">
-    <value>Математическая операция имеет неверное количество параметров</value>
+    <value>Неверное количество параметров функции</value>
```
[the remark this answers is not in the document]
Comment (CaptainDno) [reply src/Calculator/Resources/ru-RU/Resources.resw:3610]: That is true, I surely overlooked this one.
This error can be triggered by trying to input `y=log()` in graphing mode for example. But entering `y=x^` gives a different error (Непредвиденные символы в выражении). Логарифмирование is mathematical operation, however we can also say that logarithm is a function.
Maybe translation should be "Неверное количество параметров математической операции".  Long, but covers all possible cases. Or just "Неверное количество параметров".

**Comment by CaptainDno** [comment]
Sadly, the issue template for localization suggestion says that PRs with localization changes cannot be merged directly because files are generated by some internal system.
https://github.com/microsoft/calculator/blob/1bc539ffbb2d0562192d74c3c459657d7ed8d9d1/.github/ISSUE_TEMPLATE/localization_suggestion.md?plain=1#L10

Thanks for all your corrections. I will post a comment to the original issue to make sure that they are not ignored.

**Comment by CaptainDno** [comment]
Changes were merged in #2149.
Then some strings were edited again in #2181. I can't see any reason why `Область определения` was changed to more ambiguous `Область`, but in any case, I think this PR can be closed now. @BuiIdinger

**Comment by grochocki** [comment]
Unfortunately, we're not able to merge translation suggestions in this repo because of limitations in our tooling. I have filed an issue internally to track this, and our localization team will review and apply these changes. Thank you for your contribution!

**Pull request closed without merging**